beta/components/header/index.js

```diff
@@ -344,83 +341,82 @@ class Header extends Component {
             </li>
           </ul>
         </nav>
-        ${subMenu()}
+        ${this.renderSearch()}
+        ${menu}
       </header>
     `
   }
 
   renderSubMenuItems ({ name = 'library', eventName }, machine) {
-    return () => {
```

Comment:
I didn't see a reason for why `renderSubMenuItems` should return a function, so I simplified that away.